docs/CHANGELOG.rst

```diff
@@ -11,6 +11,7 @@ jax 0.2.10 (Unreleased)
 -----------------------
 * `GitHub commits <https://github.com/google/jax/compare/jax-v0.2.9...master>`__.
 * New features:
+  * :func: `jax.scipy.stats.chi2` is now available as a distribution with logpdf and pdf methods.
```

Comment:
This will not render correctly with the space. Please change to
```
:func:`jax.scipy.stats.chi2`
```

Reply:
Will do, thanks!